assets/shaders/custom_phase_item.wgsl

```diff
@@ -0,0 +1,37 @@
+// `custom_phase_item.wgsl`
+//
+// This shader goes with the `custom_phase_item` example. It demonstrates how to
+// enqueue custom rendering logic in a `RenderPhase`.
+
+// The GPU-side vertex structure.
+struct Vertex {
+    // The world-space position of the vertex.
+    @location(0) position: vec3<f32>,
+    // The color of the vertex.
+    @location(1) color: vec3<f32>,
+};
+
+// Information passed from the vertex shader to the fragment shader. (The name
+// comes from OpenGL.)
+struct Varyings {
```

Comment:
I think this should be either `FragmentInput` or `VertexOutput` for consistency with the other examples and bevy's internals.